compiler/rustc_errors/messages.ftl

```diff
@@ -1,3 +1,9 @@
+errors_expected_lifetime_parameter =
+    expected lifetime {$count ->
+        [1] -> parameter
+        *[other] -> parameters
```

Comment:
```suggestion
        [1] parameter
        *[other] parameters
```

There shouldn't come any arrow after the selector. As it stands, the `->` is taken literally meaning it's gonna show up in the actual diagnostic as shown to the user.

Reply:
Oh, I've just confused with other language on writing that lol